web/source/build.sh

```diff
@@ -83,7 +83,8 @@ minifier="$CLOSURECOMPILERPATH/compiler.jar"
 # `jsDocMissingType` prevents errors on type documentation Closure thinks is missing.  TypeScript may not
 # have the same requirements, and we trust TypeScript over Closure.
```

Comment:
Maybe add a short note to document the closure issues with older Android API levels. 
Then when we deprecate API 19, maybe we can safely revert this